pkg/Library/NewMedia/image-resource.js

```diff
@@ -46,9 +49,9 @@ export class ImageResource extends Xen.Async {
     this.canvas = this._dom.$('canvas');
     this.canvasId = Resources.allocate(this.canvas);
   }
-  // shouldUpate() {
-  //   return !this.updating;
-  // }
+  _wouldChangeProp(name, value) {
```

Comment:
i don't remember what this does, can you, please, remind me 🙏

Reply:
XenState objects call this function when a property is set to determine if the state should be invalidated.  The original notion was to support both immutable and mutable conventions by letting wouldChange*() decide whether to compare references or use deepEqual (expensive).